### PR TITLE
adds implementation for adding operations to db loader

### PR DESCRIPTION
**Pull request by jorgeantonio21**
It provides an initial interface to apply changes (delete, insert, update) to a db, via the DB loader. For more information, take a look at issue #32.

### CODE REVIEW

**Comment by jorgeantonio21** [comment]
Also took the opportunity to clean up clippy warnings.

**Comment by benjaminbollen** [comment]
`match` is not exhaustive the build says

**Comment by jorgeantonio21** [comment]
resolves #32

File: eureka-sink-postgres/db/src/ops.rs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+use crate::{
+    db_loader::Loader,
+    error::DBError,
+    operation::{Operation, OperationType},
+    sql_types::SqlType,
+};
+
+#[allow(dead_code)]
+impl Loader {
+    pub fn insert(
```
Comment (benjaminbollen) [inline eureka-sink-postgres/db/src/ops.rs:12]: 🙌

**Review by benjaminbollen — APPROVED** [review]
LGTM

**Pull request merged**